INSTALLING.md

```diff
@@ -1,98 +1,37 @@
 # Development setup for Constipated Koala
 
 Koala is written using the [Ruby on Rails] framework, which uses the
-programming language [Ruby]. We use [rbenv] to manage the version of Ruby that
-is used, and the package manager [Bundler] to manage dependencies.
+programming language [Ruby]. We use [Nix] to manage dependencies.
+(Also see the [dependency managment readme](/DEPENDENCIES.md)).
 Rails is based on the Model-View-Controller paradigm, and has an easy to
 understand file structure. On the production server, we use [Unicorn] to run
 the application with multiple threads, and [Nginx] as a proxy.
 In both development and production, we use [Mariadb] as the database.
 
-[Bundler]: https://bundler.io
+[Nix]: https://nixos.org/
 [Mariadb]: https://mariadb.org
 [Ruby on Rails]: https://guides.rubyonrails.org/getting_started.html
 [Ruby]: https://www.ruby-lang.org/
 [Unicorn]: https://bogomips.org/unicorn/
-[rbenv]: https://github.com/rbenv/rbenv
 
 ## Requirements
 
 To get started, you will need:
 
-- A Linux installation (we assume you're using Ubuntu 18.04)
+- Preferably a Linux installation, but MacOS or WSL2 should work too
 - Git (`sudo apt install git`)
-
-You'll install:
-
-- Dependencies to build Ruby and some extensions,
-- [rbenv], a Ruby version manager,
-- [Ruby] itself,
-- [Yarn], a JavaScript package manager,
-- Koala's dependencies.
+- Nix (<https://nixos.org/download.html>)
 
 To start, clone the project:
 
 ```console
 git clone git@github.com:svsticky/constipated-koala.git koala.svsticky.nl
 cd koala.svsticky.nl
+nix-shell
```

Comment:
It would be nice to mention the setup of Cachix here:

```suggestion
nix-env -iA cachix -f https://cachix.org/api/v1/install
cachix use svsticky-constipated-koala
nix-shell
```

You also get nicer highlighting when using `console` code blocks if you prepend `$` to your commands, e.g.

```console
$ foo --bar=asdf
output of that command

# foo --bar=asdf
output of that command
```